File: eTeatar/eTeatar.Model/Termin.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class Termin
    {
        public int TerminId { get; set; }

        public DateTime Datum { get; set; }

        public string Status { get; set; }

        public int DvoranaId { get; set; }

        public int PredstavaId { get; set; }

        public virtual ICollection<Rezervacija> Rezervacijas { get; set; } = new List<Rezervacija>();
    }
}

File: eTeatar/UI/eteatar_mobile/lib/models/termin.dart
import 'package:eteatar_mobile/models/predstava.dart';
import 'package:json_annotation/json_annotation.dart';

part 'termin.g.dart';

@JsonSerializable()
class Termin {
  int? terminId;
  DateTime? datum;
  String? status;
  int? predstavaId;
  int? dvoranaId;
  Predstava? predstava;

  Termin(
    this.terminId,
    this.datum
  );

  factory Termin.fromJson(Map<String, dynamic> json) => _$TerminFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$TerminToJson(this);
}